internal/backends/sqlite/metadata/registry.go

```diff
@@ -49,6 +52,17 @@ const (
 	subsystem = "sqlite_metadata"
 )
 
+var (
+	// ErrDatabaseDoesNotExist indicates that database does not exist.
+	ErrDatabaseDoesNotExist = fmt.Errorf("database does not exist")
+
+	// ErrCollectionDoesNotExist indicates that collection does not exist.
+	ErrCollectionDoesNotExist = fmt.Errorf("collection does not exist")
+
+	// ErrCollectionAlreadyExists indicates that collection already exists.
+	ErrCollectionAlreadyExists = fmt.Errorf("collection already exists")
+)
+
```

Comment:
Please remove all those errors. The complicated and incomplete error translation is one of the biggest problems of pgdb, and we should not make the same mistake